openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -152,7 +164,12 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 		return err
 	}
 	if targetMember == nil {
```

Comment:
style prefers switch/case when we have more than two choices and it's a "one of" situation

```go
switch{
case err != nil:
    return err

case targetMember == nil && memberDirExists:
    // we weren't able to locate other members and need to return based previous memberDir so we can restart.  This is the off and on again flow.
    fmt.Printf(o.TargetName)
    return nil

case targetMember == nil && !memberDirExists:
    // our member has not been added to the cluster and we have no previous data to start based on.
    return fmt.Errorf("timed out")

case targetMember != nil && len(targetMember.Name) == 0  && memberDirExists:
    // our member has been added to the cluster and has never been started before, but a data directory exists. This means that we have dirty data we must remove
    archiveDataDir(filepath.Clean(o.DataDir))

default:
    // a target member was found, but no exception circumstances.
}
```